.changeset/unlucky-beers-agree.md

```diff
@@ -0,0 +1,11 @@
+---
+"@aws-amplify/ui-react": patch
+---
+
+fix(ui-react): Collection search noResultsFound
+
+As an Amplify UI app customer, when my search in a Collection component returns nothing, I should see the message "No results found" or custom text.
```

Comment:
I don't think our agile user story language should leak into our changelog notes. This should be short, sweet, to the point, and include an example.

See "feat: adding formatValue prop on SliderField to allow formatting the value" example in changelog:
https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CHANGELOG.md